diceware/generate.go

```diff
@@ -137,21 +149,34 @@ func WordAt(i int) string {
 }
 
 // RollDie rolls a single 6-sided die and returns a value between [1,6].
+//
+// Deprecated: Use Generator.RollDie instead
```

Comment:
What's the goal with deprecating the top-level functions?

Reply:
Ah sorry. I was trying to deprecate the top-level wordlist functions, but I want to keep the "default generator" top-level functions.

Reply:
Gotcha. Are you okay with what's there so long as it's not marked deprecated?

Reply:
👍

Reply:
Pushed changes